src/mkdocstrings_handlers/python/handler.py

```diff
@@ -239,8 +240,9 @@ def collect(self, identifier: str, config: Mapping[str, Any]) -> CollectorItem:
                 loader.load_module(module_name)
             except ImportError as error:
                 raise CollectionError(str(error)) from error
-
-            unresolved, iterations = loader.resolve_aliases(implicit=False, external=False)
+            unresolved, iterations = loader.resolve_aliases(
+                implicit=False, external=config.get("load_external_modules", False)
```

Comment:
Lets use `final_config` for consistency. This way the key always appear in the dict, defaulting to false.

```suggestion
                implicit=False, external=final_config["load_external_modules"]
```